docs/source/tutorials/javascript.md

```diff
@@ -0,0 +1,300 @@
+# Building a JavaScript Application
+
+In this tutorial, you’ll build a browser-based JavaScript application that detects objects in images using Transformers.js. The app requires no server, external libraries, or build tools. All you need is a code editor and a browser.
+
+Here's how it works: the user uploads an image to the browser via the “Upload image” button. The app then downloads an AI model, runs the image through it, and finally displays an overview of where the different objects are located, like this:
+
+![Demo](https://huggingface.co/datasets/Xenova/transformers.js-docs/resolve/main/js-detection-interence-zebra.png)
+
+**************************Useful links:**************************
+
+- [Demo site](https://huggingface.co/spaces/Scrimba/javascript-object-detector)
+- [Source code walk through (scrim)](https://scrimba.com/scrim/co87f4ae7aea87a94a732f298)
```

Comment:
```suggestion
- [Interactive code walk-through (scrim)](https://scrimba.com/scrim/co87f4ae7aea87a94a732f298)
```

I just also want to highlight how amazing this interactive video/code walk-though is! 🤗 
As discussed over slack, I'll suggest a few edits to the source code, but the functionality, explanation and demonstration you recorded is perfect! You mentioned it hopefully won't be too much effort to re-record, and after this revision I don't think we would need to make any more changes.

A small correction in the video: the model is 43 MB, not 20MB.